Makefile

```diff
@@ -101,6 +101,9 @@ test: jsonnet libjsonnet.so libjsonnet_test_snippet libjsonnet_test_file
 	cd test_suite ; ./run_tests.sh
 	cd test_suite ; ./run_fmt_tests.sh
 
+reformat:
```

Comment:
I don't think make all should change source code.  However it can run the test-formatting.